src/components/augmentedReality/AugmentedRealityView.js

```diff
@@ -0,0 +1,122 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  Viro3DObject,
+  ViroAmbientLight,
+  ViroARImageMarker,
+  ViroARScene,
+  ViroARTrackingTargets,
+  ViroSound
+} from '@viro-community/react-viro';
+
+import { texts } from '../../config';
+
+export const AugmentedRealityView = ({ sceneNavigator }) => {
+  const {
+    isObjectLoading,
+    setIsObjectLoading,
+    isStartAnimationAndSound,
+    setIsStartAnimationAndSound,
+    object
+  } = sceneNavigator.viroAppProps;
+
+  // TODO: these data can be updated according to the data coming from the server when the
+  //       real 3D models arrive
+  const position = [0, 0, 0];
+  const rotation = [0, 0, 0];
+  const scale = [0.002, 0.002, 0.002];
```

Comment:
why did the scale change from 0.02 to 0.002? isn't it nearly 0 now and 0 would work the same?

Reply:
how can we see if we make the size of the object 0? Yes, this is close to 0, but not 0. 0.02 was updated to 0.002 because it was too big. If I give it 0, it will not appear on the screen. If you delete scale, you will see how big the default size of the object is.

Reply:
ah ja ok i see. so we would need to give the artists some information about the size of the arts in order to not have to set a scale here? or to just have a scale here for being set in order it is still too big.
is there some value we can give the artists to not provide too big arts?

Reply:
Unfortunately, I don't know how this value is determined.

Reply:
can you find an object where no scale is needed? than we could send this to artists as template?

Reply:
I guess I'll have to try a lot of objects. I'm not sure I can find this one.

Reply:
ok